docs/content/docs/advanced/good-practices.md

```diff
@@ -24,16 +24,18 @@ To remove the default config, you can replace the ``eslintConfig`` in your
 ```json
 {
   "eslintConfig": {
-    "parser": "babel-eslint",
     "extends": "eslint:recommended",
-    "ecmaFeatures": {
-      "modules": true,
-      "experimentalObjectRestSpread": true
+    "parserOptions": {
+      "ecmaVersion": 6,
+      "sourceType": "module",
+      "ecmaFeatures": {
+        "experimentalObjectRestSpread": true,
```

Comment:
> Default boilerplate is using package.json to set eslint config.

Yeah, so what is the problem? This place is for people that want minimal eslint config (not the shareable one)